pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -444,6 +448,9 @@ func BootstrapKuryr(conf *operv1.NetworkSpec, kubeClient client.Client) (*bootst
 	log.Print("Creating OpenShift API loadbalancer pool members")
 	r, _ := regexp.Compile(fmt.Sprintf("^%s-(master-port-[0-9]+|bootstrap-port)$", clusterID))
 	portList, err := listOpenStackPortsMatchingPattern(client, tag, r)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to list ports with matching pattern")
```

Comment:
"failed to get OpenShift master nodes ports"

Reply:
failed to get OpenShift master or bootstrap nodes ports